### ISSUE TITLE
JS/TS Rule direct-response-write generates a false-positive when using AWS sdk commands

### ISSUE THREAD

**Issue by ctooley21** [returntocorp/sgrep-rules#2872]
**Describe the bug**
The rule direct-response-write generates a false positive when using a AWS sdk command within an express route.

**To Reproduce**
```
import {
    AdminUpdateUserAttributesCommand,
    CognitoIdentityProviderClient,
} from "@aws-sdk/client-cognito-identity-provider";
import express from "express";

const router = express.Router();

const client = new CognitoIdentityProviderClient({
    region: "us-east-1",
});

router.post(
    "/:id",
    async (req, res, next) => {
        const userId = req.params?.id;

        if (user.email !== req.body.email) {
            const command = new AdminUpdateUserAttributesCommand({
                Username: user.cognitoUserId,
                UserPoolId: process.env.COGNITO_USER_POOL_ID,
                UserAttributes: [
                    {
                        Name: "email",
                        Value: req.body.email,
                    }
                ],
            });

            await client.send(command);
        }

        res.status(200).send();
    }
);
```

**Expected behavior**
The rule should not be triggered when using AWS sdk commands

**Priority**
How important is this to you?
- [ ] P0: blocking me from making progress
- [ ] P1: this will block me in the near future
- [x] P2: annoying but not blocking me

**Additional Context**
